test/e2e/framework/create.go

```diff
@@ -54,7 +54,7 @@ func (f *Framework) LoadFromManifests(files ...string) ([]interface{}, error) {
 	err := visitManifests(func(data []byte) error {
 		// Ignore any additional fields for now, just determine what we have.
 		var what What
-		if err := runtime.DecodeInto(legacyscheme.Codecs.UniversalDecoder(), data, &what); err != nil {
```

Comment:
I'm a bit unsure about this change, is it safe to remove legacyscheme here? @liggitt